app/models/event.rb

```diff
@@ -0,0 +1,16 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :event_attending
+  has_many :attendees, through: :event_attending, source: :user
+
+  def self.past_event
+    where('date < ?', Date.today)
+  end
+
+  def self.upcoming_event
+    where('date >= ?', Date.today)
+  end
```

Comment:
Since we already have scope for checking upcoming and past events, this will not be necessary. We can get rid of it.